.github/ISSUE_TEMPLATE/feedback.yml

```diff
@@ -1,12 +1,12 @@
-name: FinOps Use Case Feedback
-description: Provide feedback on FinOps use cases that cannot be performed with the current FOCUS specification.
+name: Feature Request and Use Case Feedback
+description: Provide feedback on unsupported use cases or features in the FOCUS specification, including FinOps scenarios, to help prioritize updates. Avoid sharing proprietary information.
 title: "[FEEDBACK]: "
```

Comment:
Shall we perhaps standardize on Proper Case instead of CAPS for the prefixes...?

Reply:
Yes, correct.  How about `[Feedback]` instead?